samples/getting-started/DemoApp/main.swift

```diff
@@ -9,4 +9,5 @@
 import UIKit
 import Foundation
 
-UIApplicationMain(0, nil, nil, NSStringFromClass(AppDelegate.self))
+var argv = [UnsafeMutablePointer<Int8>]()
+_ = UIApplicationMain(0, &argv, nil, NSStringFromClass(AppDelegate.self))
```

Comment:
@michaelknoch what's the reason we now must pass an empty `argv` instead of `nil`?

Reply:
Its because of this deprecation warning
<img width="881" alt="screenshot 2018-12-26 at 13 36 18" src="https://user-images.githubusercontent.com/5617793/50446203-089ccb00-0914-11e9-96ae-93c2c0d5df15.png">

Reply:
boah danke, finally!

Reply:
@rikner aside from the deprecation, the original reason I made the change on iOS is because iOS now crashes on this call if you pass `nil`, so we have to pass something. But we can just pass CommandLine.unsafeArgv now